local/kcp/start.sh

```diff
@@ -40,7 +40,7 @@ kcp-binaries() {
     git clone https://github.com/kcp-dev/kcp.git
     KCP_DIR="${KCP_PARENT_DIR}/kcp"
     pushd kcp
-    KCP_BRANCH="${KCP_BRANCH:-release-0.5}"
+    KCP_BRANCH="${KCP_BRANCH:-$(yq '.images[] | select(.name == "kcp") | .newTag' "$PARENT_PATH/../../ckcp/openshift/overlays/dev/kustomization.yaml")}"
```

Comment:
this change reduces readability and creates unnecessary dependencies, hence fragility.
If you don't want to change the default version in multiple places let's have a file with them.

Reply:
I tried to improve readabilty.

I'm not sure I want to start refactoring/implementing a single source of truth for version with this PR, but I agree that it would be nice to have at some point, so I've logged it in PLNSRVCE-498.